src/SourceGenerators/Uno.UI.SourceGenerators.net6.Tests/Windows_UI_Xaml_Controls/GridTests/Given_Grid.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Uno.UI.SourceGenerators.Tests.Windows_UI_Xaml_Controls.GridTests;
+
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis;
+using static XamlCodeGeneratorHelper;
+
+[TestClass]
+public class Given_Grid
+{
+	[TestMethod]
+	public async Task When_Grid_Uses_Both_Syntaxes()
```

Comment:
Great! The tests appear in the tests output: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=53546&view=ms.vss-test-web.build-test-results-tab&runId=1937972&resultId=102154&paneView=debug